monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -47,14 +47,14 @@ groups:
   - name: osd
     rules:
       - alert: 10% OSDs down
-        expr: (sum(ceph_osd_up) / count(ceph_osd_up)) * 100 <= 90
+        expr: count(ceph_osd_up == 0) / count(ceph_osd_up) >= .1
         labels:
           severity: critical
           type: ceph_default
           oid: 1.3.6.1.4.1.50495.15.1.2.4.1
         annotations:
           description: |
-            {{ $value | humanize}}% or {{with query "sum(ceph_osd_up)" }}{{ . | first | value }}{{ end }} of {{ with query "count(ceph_osd_up)"}}{{. | first | value }}{{ end }} OSDs are down (>=10%).
+            {{ $value | humanizePercentage }} or {{ with query "count(ceph_osd_up == 0)" }}{{ . | first | value }}{{ end }} of {{ with query "count(ceph_osd_up)" }}{{ . | first | value }}{{ end }} OSDs are down (≥ 10%).
```

Comment:
@BenoitKnecht It works! But ```humanizePercentage``` does not exist for prometheus versiones <2.11.
In order to guarantee backward compatibility, do you mind to replace for another equivalent expression ?
It shouldn't be difficult.... @epuertat @p-se @LenzGr Thoughts about it?

Reply:
Sure, I can change it. But should it be done on `master`, or only in the backports for older Ceph versions? I don't know if there's minimum Prometheus version that is supposed to be supported for each Ceph release, or if we should try to support the broadest set possible.

For reference, Prometheus 2.11.0 came out in July 2019.

Reply:
I don't think it's an issue for pacific, but as discussed, let's have that conversation tomorrow in either our standup or the backlog grooming meeting.

Reply:
We've had the discussion on IRC and it is unlikely that Ceph Pacific (current master) will only be supported on containers. This results in doubts, as we will need to support a wide range of distributions with possibly older Prometheus versions. Hence we concluded to proceed with the most compatible approach.

Can you please adapt the code to not use `humanizePercentage`?

Reply:
Done.